### PR TITLE
[3.6] Add manage suggestions link into browse-all metabox footer

### PR DESCRIPTION
**Pull request by haszari**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:
Adds a "Manage suggestions" link to the product list metabox, linking to settings.

### How to test the changes in this Pull Request:
1. Edit a product. Scroll down to metabox, click `Get more options` tab.
2. Notice new `Manage suggestions` link at bottom right.

Note that the link destination (the new settings section) is implemented in separate PR #23218 .

Screen Shot 2019-04-08 at 9 22 37 AM

### CODE REVIEW

File: assets/js/admin/marketplace-suggestions.js [hunk 1]
```diff
@@ -170,6 +170,26 @@
 				container.appendChild( body );
 			}
 
+			// Conditionally add in a Manage suggestions link to product edit
+			// metabox footer (based on suggestion slug).
+			var slugsWithManage = [
+				'product-edit-empty-footer-browse-all',
```
Comment (haszari) [inline assets/js/admin/marketplace-suggestions.js]: Note that I've added another alias for this suggestion, as the original one was misleading / confusing. I'm renaming it in the json data, but I've got both here. In future we can remove `product-edit-empty-footer-browse-all`.

**Comment by peterfabian** [comment]
Similarly to @kloon 's point over at #23218, should this link be shown only to admins/users who actually can change this setting to prevent confusion?

**Comment by timmyc** [comment]
> Similarly to @kloon 's point over at #23218, should this link be shown only to admins/users who actually can change this setting, to prevent confusion?

@peterfabian the suggestions [areas are only shown](https://github.com/woocommerce/woocommerce/blob/master/includes/admin/marketplace-suggestions/class-wc-marketplace-suggestions.php#L177) to users with the capabilities to install plugins, so these links will only be present when the suggestions are visible.

**Review by timmyc — APPROVED** [review]
Thanks Rua, this is testing out well for me. Let's coordinate merges to navigate all the merge conflicts accordingly.

**Pull request merged**